packages/sdk/src/constants.ts

```diff
@@ -24,5 +24,19 @@
 
  */
 
+import { IAuthSession, functionalSdk as rtlSdk } from '@looker/sdk-rtl'
+
 export const sdkVersion = '21.4'
 export const environmentPrefix = 'LOOKERSDK'
+
+/**
+ * Creates an "sdk" to be used with the Typescript SDK's funcs.ts file
+ * @param authSession authentication session
+ * @param apiVersion version of API to use (e.g. "3.1" or "4.0")
+ */
+export const functionalSdk = (
```

Comment:
Seems strange to have this in `constants.ts`

Again, add two methods here for `functionalSdk31` and `functionalSdk40`?